cirq-core/cirq/sim/classical_simulator_test.py

```diff
@@ -17,180 +17,191 @@
-        expected_results = {'key': np.array([[[1, 0], [0, 0]]], dtype=np.uint8)}
-        sim = cirq.ClassicalStateSimulator()
-        results = sim.run(circuit, param_resolver=None, repetitions=1).records
-        np.testing.assert_equal(results, expected_results)
-
-    def test_repetitions(self):
-        q0 = cirq.LineQubit.range(1)
-        circuit = cirq.Circuit()
-        circuit.append(cirq.measure(q0, key='key'))
-        expected_results = {
-            'key': np.array(
-                [[[0]], [[0]], [[0]], [[0]], [[0]], [[0]], [[0]], [[0]], [[0]], [[0]]],
-                dtype=np.uint8,
-            )
-        }
-        sim = cirq.ClassicalStateSimulator()
-        results = sim.run(circuit, param_resolver=None, repetitions=10).records
-        np.testing.assert_equal(results, expected_results)
-
-    def test_multiple_gates(self):
-        q0, q1 = cirq.LineQubit.range(2)
-        circuit = cirq.Circuit()
-        circuit.append(cirq.X(q0))
-        circuit.append(cirq.CNOT(q0, q1))
-        circuit.append(cirq.CNOT(q0, q1))
-        circuit.append(cirq.CNOT(q0, q1))
-        circuit.append(cirq.X(q1))
-        circuit.append(cirq.measure((q0, q1), key='key'))
-        expected_results = {'key': np.array([[[1, 0]]], dtype=np.uint8)}
-        sim = cirq.ClassicalStateSimulator()
-        results = sim.run(circuit, param_resolver=None, repetitions=1).records
-        np.testing.assert_equal(results, expected_results)
-
-    def test_multiple_gates_order(self):
-        q0, q1 = cirq.LineQubit.range(2)
-        circuit = cirq.Circuit()
-        circuit.append(cirq.X(q0))
-        circuit.append(cirq.CNOT(q0, q1))
-        circuit.append(cirq.CNOT(q1, q0))
-        circuit.append(cirq.measure((q0, q1), key='key'))
-        expected_results = {'key': np.array([[[0, 1]]], dtype=np.uint8)}
-        sim = cirq.ClassicalStateSimulator()
-        results = sim.run(circuit, param_resolver=None, repetitions=1).records
-        np.testing.assert_equal(results, expected_results)
-
-    def test_param_resolver(self):
-        gate = cirq.CNOT ** sympy.Symbol('t')
-        q0, q1 = cirq.LineQubit.range(2)
-        circuit = cirq.Circuit()
-        circuit.append(cirq.X(q0))
-        circuit.append(gate(q0, q1))
-        circuit.append(cirq.measure((q1), key='key'))
-        resolver = cirq.ParamResolver({'t': 0})
-        sim = cirq.ClassicalStateSimulator()
-        results_with_parameter_zero = sim.run(
-            circuit, param_resolver=resolver, repetitions=1
-        ).records
-        resolver = cirq.ParamResolver({'t': 1})
-        results_with_parameter_one = sim.run(
-            circuit, param_resolver=resolver, repetitions=1
-        ).records
-        np.testing.assert_equal(
-            results_with_parameter_zero, {'key': np.array([[[0]]], dtype=np.uint8)}
-        )
-        np.testing.assert_equal(
-            results_with_parameter_one, {'key': np.array([[[1]]], dtype=np.uint8)}
+def test_x_gate():
```

Comment:
Is this just removing the class or are there other changes buried in here?  It's hard to tell with the git diff.